WordPress/src/main/java/org/wordpress/android/ui/posts/editor/EditorActionsProvider.kt

```diff
@@ -24,23 +25,15 @@ class EditorActionsProvider @Inject constructor(private val remoteLoggingUtils:
                 PostStatus.PENDING,
                 PostStatus.UNKNOWN -> PrimaryEditorAction.SUBMIT_FOR_REVIEW
                 PostStatus.TRASHED -> {
-                    // TODO if this log doesn't appear in Sentry, we should start throwing IllegalStateException
-                    //  instead of returning a valid action.
-                    remoteLoggingUtils.log(
-                            "User shouldn't be able to open a trashed post in an editor " +
-                                    "without publishing rights."
-                    )
+                    AppLog.e(T.EDITOR, "User shouldn't be able to open a trashed post in an editor " +
```

Comment:
IMHO if we're concerned about this, we should either crash if we're in an invalid state, OR we could just do the right thing, but for now we have the log.